docker/templates/README.md

```diff
@@ -0,0 +1,79 @@
+# 🍪 Containerization templates
+
+This directory contains [cookiecutters](https://github.com/audreyr/cookiecutter)
+for the containerization of web applications.
+
+## Getting started
+
+### 1. Set up.
+
+Install `cookiecutter` in your virtual environment of choice. (It's a [great
+candidate for `gus`](https://github.com/datamade/ops/wiki/How-to-DataMade-(Resources-for-new-hires)#python)!)
+
+```bash
+pip install cookiecutter
+```
+
+Then, clone this repository to your favorite working directory –
+
+```bash
+git clone https://github.com/datamade/how-to.git
+```
+
+– or update your local version.
+
+```bash
+cd /path/to/how-to
+git pull
+```
+
+### 2. Run `cookiecutter`.
+
+`cd` into your project directory. From here, we'll use `cookiecutter` templates
+to generate deployment scripts for your app or server.
+
+From your project directory, run `cookiecutter`:
+
+```bash
+cookiecutter how-to/docker/templates/python
+```
+
+#### Template variables
+
+When you run `cookiecutter`, it will ask you to define the following variables
+in your terminal:
+
+| Variable | Definition |
+| - | - |
+| `directory_name` | The directory that will contain your generated files. We'll move the files out and remove this directory after running `cookiecutter` so it's fine to use the default here. |
+| `app_name` | The name of your application. |
+| `local_settings` | If your project includes a local settings file, set this equal to the relative path to your local settings file (or your example settings file, if it includes working values), and it will be automatically mounted into your application container. Set this to None if your application does not use a local settings file. |
```

Comment:
I mistakenly left this empty instead of setting it to the string `None` and was surprised to realize that it accepted the default value instead 😅 A bozo move, but one that I wouldn't be surprised to see other people make too.

```suggestion
| `local_settings` | If your project includes a local settings file, set this equal to the relative path to your local settings file (or your example settings file, if it includes working values), and it will be automatically mounted into your application container. Set this to the string `None` if your application does not use a local settings file. |
```